tests/unit/__snapshots__/visitor-keys.js.snap

```diff
@@ -1022,6 +1022,9 @@ exports[`visitor keys estree 1`] = `
     "expression",
     "typeAnnotation",
   ],
+  "TypeOperator": [
+    "typeAnnotation",
+  ],
```

Comment:
We need to support this after merging https://github.com/prettier/prettier/pull/15455 ( `typeof MyGenericClass<string, number>` )

Reply:
I removed unsupported nodes in previous PR, https://github.com/prettier/prettier/blob/84a5faf7a82dbe72030707315d058d15f9e77066/src/language-js/traverse/visitor-keys.evaluate.js#L86 but not important.